content/docs/querying/functions.md

```diff
@@ -80,6 +86,54 @@ and value across all series in the input vector.
 `floor(v instant-vector)` rounds the sample values of all elements in `v` down
 to the nearest integer.
 
+## `histogram_quantile()`
+
+`histogram_quantile(φ float, b instant-vector)` calculates the
```

Comment:
Surely we can do that, but then we have to write a line further down that `d` stands for the φ in "φ-quantile". It's an additional layer of indirection. I can't see how that will help to reduce confusion.